### PR TITLE
Add the API url for Girder inside of Celery headers

### PR DESCRIPTION
**Pull request by danlamanna**
@zachmullen

### CODE REVIEW

File: plugins/worker/server/utils.py [hunk 1]
```diff
@@ -134,9 +134,12 @@ def jobInfoSpec(job, token=None, logPrint=True):
     if isinstance(token, dict):
         token = token['_id']
 
+    apiUrl = getWorkerApiUrl()
+
     return {
         'method': 'PUT',
-        'url': '/'.join((getWorkerApiUrl(), 'job', str(job['_id']))),
+        'url': '/'.join((apiUrl, 'job', str(job['_id']))),
+        'apiUrl': apiUrl,
```
Comment (manthey) [inline plugins/worker/server/utils.py]: We call this api_url in the `girderInputSpec` and `girderOutputSpec` functions.

**Comment by zachmullen** [comment]
This integrates with https://github.com/girder/girder_worker/pull/104 if I understand correctly.

This isn't really to do with job management per se, right? Does it belong in jobInfoSpec?

File: plugins/worker/server/utils.py (hunk 1 above)
Comment (danlamanna) [inline plugins/worker/server/utils.py]: I either have to be inconsistent with those, or inconsistent with the keys in this returning dictionary.

**Comment by danlamanna** [comment]
> Does it belong in jobInfoSpec?

The alternative is to add the api url as a separate header [when we send tasks to celery](https://github.com/girder/girder/blob/0e4382ff43c7ccdba0a9be227945632fb99c8d54/plugins/worker/server/__init__.py#L84), maybe that's the right decision here.

The jobInfoSpec as it is exposes the URL a job should call back to to report status updates, I'm simply adding the URL of the spawning Girder instance so the job can call back for other things.

For context, keep in mind that the use case here is using Girder worker *without* input or output specs.

**Comment by zachmullen** [comment]
Separate header seems like the right choice semantically.

**Comment by danlamanna** [comment]
Okay, @manthey @zachmullen, this should be ready to review.

**Comment by zachmullen** [comment]
LGTM on the Girder side, it looks like the worker side hasn't changed to use this new header yet.

**Comment by danlamanna** [comment]
@zachmullen I probably won't get to the worker side today (though it will still continue to function fine). I just wanted to get this in under the wire for the next release.

**Pull request merged**